include/openssl/bio.h

```diff
@@ -148,12 +148,16 @@ extern "C" {
 /* internal BIO see include/internal/bio.h:
```

Comment:
Ah, this is a multi line comment, and therefore it starts with `/*` on a line of it's own,
and since it is a formatted text, it is also possible to start with `/*-`

Reply:
Done